appeals-service-api/src/controllers/appeals.js

```diff
@@ -2,12 +2,11 @@ const uuid = require('uuid');
 const logger = require('../lib/logger');
 
 const mongodb = require('../db/db');
-const { appealDocument } = require('../models/appeal');
 const { validateAppeal } = require('../middleware/validateAppeal');
 
 module.exports = {
   async create(req, res) {
-    const appeal = appealDocument;
+    const appeal = req.body;
```

Comment:
I think the current appeals api spec does not expect a payload. Here we use a known to be 'safe' state for the newly created appeal.

Reply:
The thing is that on the front I need to initialise a new appeal before, the only difference is that he has no id.
And it's weird to have to ask to create an empty skeleton and then update it. All the first task you do will be replaced with the skeleton without this change.

Reply:
I think the way to initialise a new appeal at any point is to call the POST (create) endpoint with no payload. The response will be a safe appeal json representation of an appeal with a new id. You can build and submit any PUT updates using that as a template payload.

Reply:
It was done like this because the previous model was just flat and updated on the fly. Now we have a new data model, why not change ? 

Is there a reason why we would not want to offer the capacity to insert an appeal with the body on the payload ? It seems to be the standard for POST method in REST architecture. I don't really like the 'transactional' way that you describing and the safety you mentioned should not be the issue as we are supposed to restrict the API soon and if the body in the payload is not safe, he will not be stored. 

Did I understood correctly ?